terraform/devnet-0/hetzner.tf.disabled

```diff
@@ -0,0 +1,228 @@
+////////////////////////////////////////////////////////////////////////////////////////
+//                            TERRAFORM PROVIDERS & BACKEND
+////////////////////////////////////////////////////////////////////////////////////////
+provider "hcloud" {
+  token = var.hcloud_token
+}
+////////////////////////////////////////////////////////////////////////////////////////
+//                                        VARIABLES
+////////////////////////////////////////////////////////////////////////////////////////
+variable "hcloud_token" {
+  type        = string
+  description = "Hetzner Cloud API Token"
+  sensitive   = true
+}
+
+variable "hcloud_ssh_key_fingerprint" {
+  type    = string
+  default = "d6:76:2d:9c:5b:33:80:ff:0f:09:a2:10:9b:58:7e:dc"
```

Comment:
guessing this is your key fingerprint? should be the shared one that's default if anything :D

Reply:
this is the shared key, copied from whisks